practice-one/src/components/Button/Button.jsx

```diff
@@ -0,0 +1,68 @@
+import React from "react";
+import "./Button.css";
+import PropTypes from "prop-types";
+
+const Button = ({
+  children,
+  width,
+  height,
+  color,
+  background,
+  fontSize,
+  fontWeight,
+  icon,
+  border,
+  align,
+}) => {
+  const buttonClasses = `btn ${color} ${background} ${border} ${fontSize} ${fontWeight} ${align}`;
+  const buttonSizes = {
+    "--btn-width": width,
+    "--btn-height": height,
+  };
+  return (
+    <button
+      type="button"
```

Comment:
why need define type="button" in here?

Reply:
Because different browsers may use different default types for the <button> element.
So I set type button for it like default
But if I use a button for form, I will set its type submit or reset.